services/venice-controller/src/main/java/com/linkedin/venice/pushmonitor/AbstractPushMonitor.java

```diff
@@ -961,10 +963,12 @@ protected void checkWhetherToStartEOPProcedures(OfflinePushStatus offlinePushSta
               activeActiveRealTimeSourceKafkaURLs);
           newStatusDetails.append("kicked off buffer replay");
         } else if (!offlinePushStatus.getCurrentStatus().isTerminal()) {
-          LOGGER.info(
-              "{} is not ready to start buffer replay. Current state: {}",
-              offlinePushStatus.getKafkaTopic(),
-              offlinePushStatus.getCurrentStatus().toString());
+          if (!REDUNDANT_EXCEPTION_FILTER.isRedundantException(offlinePushStatus.getKafkaTopic())) {
```

Comment:
Why do we need to rate limit this?

Reply:
for hybrid stores it was spamming controller log